### PR TITLE
Add realistic sample emails

### PR DESCRIPTION
**Pull request by IshaDakota**
#10

### CODE REVIEW

**Comment by IshaDakota** [comment]
fireshot capture 139 - elm - http___127 0 0 1_3000_

**Comment by IshaDakota** [comment]
@amitaibu how do we get line breaks to render in the body strings?

File: src/elm/Pages/Inbox/View.elm [hunk 1]
```diff
@@ -223,20 +217,27 @@ viewSelectedEmail model =
                             , div [ class "content__messages__selected__content" ]
                                 [ text email.body
                                 ]
+                            , div [ class "ui form icon message content__messages__action" ]
```
Comment (amitaibu) [inline src/elm/Pages/Inbox/View.elm]: See, you are already writing Elm!

**Comment by amitaibu** [comment]
I'll open a PR to fix the line breaks.

**Pull request merged**